docs/classad-attributes/job-classad-attributes.rst

```diff
@@ -228,9 +228,14 @@ all attributes.
     command that created this job.  Note this may be different than the
     version of the HTCondor daemon that runs the job.
 
-:classad-attribute-def:`ContainerImage`
-    For Container universe jobs, the string that names the container image to be run
-    the job in.
+:classad-attribute-def:`ContainerImageSource`
+    For Container universe jobs, the string that names the container image source
+    Is "local" for non-transfered images or "cedar" for transfered files.  "docker"
+    or "http" might be other common values.
+
+:classad-attribute-def:`ContainerImageTransferred`
+    For Container universe jobs, true if the container image is being transferred
```

Comment:
this should be deleted.   the patch does not add `ContainerImageTransferred` as a new job attribute anymore.